### ISSUE TITLE
Jungle mobs please!

### ISSUE THREAD

**Issue by Ninjayatsu** [AlexModGuy/Ice-and-Fire-Suggestions#785]
I Would LOVE if you guys could add more jungle mobs similar to the ampithere, I think similar styles to a mayan, or aztec like mobs would work and there is def not enough jungle mobs, maybe you could even add like a mayan temple, where an "aincent spirit" resides.

**Comment by Comdec35000** [comment]
Yes, you should add some structures like the gorgon temple (but without the generation bugs) but it's mayan temple, witch should looks like better than the vanilla jungle temple. 
It may be like a donjon with a boss, a mayan god, at the end...
A bit like the great Zigurrat of the Magic kingdom mod
